Cargo.toml

```diff
@@ -147,3 +151,4 @@ strip = ['wasm-encoder', 'wasmparser', 'regex']
 compose = ['wasm-compose']
 demangle = ['rustc-demangle', 'cpp_demangle', 'wasmparser', 'wasm-encoder']
 component = ['wit-component', 'wit-parser', 'wast', 'wasm-encoder', 'wasmparser']
+producers = ['wasmparser', 'indexmap']
```

Comment:
Oh before I forget can you update `.github/workflows/main.yml` to test the build with just the `producers` feature? There should be a spot that all the other subcommands are tested as well.